### PR TITLE
Adds underground column to the Zone Summary table in IVRS

### PR DESCRIPTION
**Pull request by JasonGlazer**
Addresses #4606
- [X] Code style (parentheses padding, variable names)
- [X] Functional code review (it has to work!)
- [X] If defect, results of running current develop vs this branch should exhibit the fix
- [X] CI status: all green or justified
- [X] Performance: CI Linux results include performance check -- verify this
- [X] Unit Test(s)
- [X] Documentation changes in place
- [X] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: src/EnergyPlus/OutputReportTabular.cc [hunk 1]
```diff
@@ -9443,14 +9443,16 @@ namespace OutputReportTabular {
 		static Array1D< Real64 > zstArea( 4 );
 		static Array1D< Real64 > zstVolume( 4 );
 		static Array1D< Real64 > zstWallArea( 4 );
-		static Array1D< Real64 > zstWindowArea( 4 );
+		static Array1D< Real64 > zstUndWallArea(4);
+		static Array1D< Real64 > zstWindowArea(4);
```
Comment (mjwitte) [inline src/EnergyPlus/OutputReportTabular.cc]: This line an line above need space padding.

File: doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex [hunk 2]
```diff
@@ -45,7 +45,7 @@ \subsubsection{Predefined Annual Summary Reports}\label{predefined-annual-summar
 \item
   Skylight-Roof Ratio table for envelope which includes the roof area and the skylight area and the ratio of the two. This includes all surfaces with a tilt of less than 60 degrees.
 \item
-  Zone Summary includes internal load summary for each zone including area, if conditioned, volume, multipliers, gross wall area, window area, design lighting, design people, design plug and process.
+  Zone Summary includes internal load summary for each zone including area, if conditioned, volume, multipliers, above ground gross wall area, underground gross wall area, window area, design lighting, design people, design plug and process.
```
Comment (mjwitte) [inline doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex:48]: What's the standard terminology, if there is any, in std 90 or elsewhere?  "Above Ground"/"Underground" or "Below Ground" or should they be "Above Grade" and "Below Grade" or?

**Comment by nrel-bot-3** [comment]
@JasonGlazer @lgentile it has been 14 days since this pull request was last updated.

File: doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex (hunk 2 above)
Comment (JasonGlazer) [inline doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex:48]: In ASHRAE Standard 90.1-2013, the definition for "wall" includes sub-definitions for "above-grade wall" and "below-grade wall" but I am not certain what terms are most commonly used within the industry.

**Comment by JasonGlazer** [comment]
@mjwitte This one looks like it is ready for any additional review you want to do.

**Pull request merged**